### PR TITLE
fix: handle missing daemon errors

### PR DESCRIPTION
**Pull request by diasdavid**

### CODE REVIEW

**Comment by diasdavid** [comment]
@pfrazee this seems to solve your issue, want to try it out?

**Comment by pfrazee** [comment]
Yeah that solved it! Thanks

File: src/request-api.js [hunk 1]
```diff
@@ -147,6 +147,10 @@ function requestAPI (config, options, callback) {
     headers: headers
   }, onRes(options.buffer, callback))
 
+  req.on('error', (err) => {
+    callback(err)
```
Comment (dignifiedquire) [inline src/request-api.js]: you should use to avoid having double callback calls

```js
const once = require('once')
callback = once(callback)
```

Comment (diasdavid) [inline src/request-api.js]: it is already done, see above :)

Comment (dignifiedquire) [inline src/request-api.js]: :)

**Comment by pfrazee** [comment]
thanks!

**Pull request merged**